client/app/components/queries/query-editor.js

```diff
@@ -119,31 +121,43 @@ function queryEditor(QuerySnippet, $timeout) {
 
         const schemaCompleter = {
           getCompletions(state, session, pos, prefix, callback) {
-            if (prefix.length === 0 || !$scope.schema) {
+            // make a variable for the auto completion in the query editor
+            $scope.autoCompleteSchema = $scope.schema; // removeExtraSchemaInfo(
+            if (prefix.length === 0 || !$scope.autoCompleteSchema) {
               callback(null, []);
               return;
             }
 
-            if (!$scope.schema.keywords) {
+            if (!$scope.autoCompleteSchema.keywords) {
               const keywords = {};
 
-              $scope.schema.forEach((table) => {
+              $scope.autoCompleteSchema.forEach((table) => {
                 keywords[table.name] = 'Table';
-
+                // remove special characters from display schema that aren't okay in sql syntax
                 table.columns.forEach((c) => {
-                  keywords[c] = 'Column';
+                  if (c.charAt(c.length - 1) === ')') {
+                    let parensStartAt = c.indexOf('(') - 1;
+                    c = c.substring(0, parensStartAt);
+                    parensStartAt = 1; // linter complains without this line
+                  }
```

Comment:
So this removes columns that end with a closing parenthesis at the point where the opening parenthesis is? Can you elaborate which data sources have those kind of colums?

Reply:
I see, let's remove this then.